### PR TITLE
Feature/account page

### PR DESCRIPTION
**Pull request by Yurii-Kupchynskyi**
Create AccontPage

### CODE REVIEW

File: src/components/AccountComponents/AccountAvatar/AccountAvatar.tsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import "./AccountAvatar.scss"
+
+interface ImageProps{
+    image : string
+}
+
+export const AccountAvatar: React.FC<ImageProps> = ({image })=>{
```
Comment (TaniaMitnovych) [inline src/components/AccountComponents/AccountAvatar/AccountAvatar.tsx:7]:
```
export const AccountAvatar: React.FC<ImageProps> = ({ image }) => {
```
add spaces

File: src/components/AccountComponents/AccountAvatar/AccountAvatar.tsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    return(
+        <div className="avatar">                
+                 <img className="avatar__image" src={image} alt="Avatar" />
+             </div>
+    )
+}
```
Comment (TaniaMitnovych) [inline src/components/AccountComponents/AccountAvatar/AccountAvatar.tsx:13]: needed to be aligned

File: src/pages/AccountPage/AccountPage.scss [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+body{
+    background-color: $color-theme-grey;
+    box-sizing: border-box;
+}
+
+
+
+
+
+
```
Comment (OlehYavoriv) [inline src/pages/AccountPage/AccountPage.scss:11]: remove empty lines

File: src/components/AccountComponents/AccountInfo/AccountInfo.tsx [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+                        <div className="item__data">
+                        <p className="underText">Email address</p>
+                        <p>{email}</p>
+                        </div>
+                    </li>
+                </ul>
+                </div>
+             </div>
+    )
+}
```
Comment (TaniaMitnovych) [inline src/components/AccountComponents/AccountInfo/AccountInfo.tsx:39]: add spaces where it's needed, remove unnecessary empty lines and align this code properly

File: src/components/AccountComponents/AccountInfo/AccountInfo.tsx [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+export const AccountInfo: React.FC<UserInfoProps> = ({given_name,family_name,email})=>{
+    return(
+        <div className="personalData">
+                <div className="personalData__block">
+                <h3 className="personalData__title">Personal data</h3>
+                <ul className="personalData__list">
+                    <li className="list__item">
+                        <div className="list__item__icon">
+                        <UserIcon/>
```
Comment (TaniaMitnovych) [inline src/components/AccountComponents/AccountInfo/AccountInfo.tsx:18]: Is it possible to replace UserIcon with a profile photo of the current user?

File: src/components/AccountComponents/AccountSignOut/AccountSignOut.tsx [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+
+export const AccountSignOut : React.FC<{}> = ({})=>{
+    return(    
+            <div className="signOut">
+            <div className="signOut__block">
+            <h3 className="signOut__title">Account</h3>
+            <SignOutButton className="no"/>
+            </div>
+            </div>       
+    )
```
Comment (TaniaMitnovych) [inline src/components/AccountComponents/AccountSignOut/AccountSignOut.tsx:12]: align properly

File: src/pages/AccountPage/AccountPage.tsx [hunk 7]
```diff
@@ -0,0 +1,23 @@
+import "./AccountPage.scss"
+import { PageHeader } from "../../components/PageHeader"
+import { AccountAvatar,AccountInfo,AccountSignOut } from "../../components/AccountComponents";
```
Comment (TaniaMitnovych) [inline src/pages/AccountPage/AccountPage.tsx:3]: add spaces after commas

File: src/routes/Routes.tsx [hunk 8]
```diff
@@ -1,10 +1,11 @@
-import { Route, Switch } from 'react-router-dom';
-import { MainPage, PlacePage, HomePage } from '../pages';
+import {Route, Switch} from 'react-router-dom';
+import { MainPage, PlacePage, HomePage,AccountPage } from '../pages';
```
Comment (TaniaMitnovych) [inline src/routes/Routes.tsx:2]: add spaces inside the braces

File: src/components/AccountComponents/AccountInfo/AccountInfo.tsx (hunk 5 above)
Comment (TaniaMitnovych) [reply src/components/AccountComponents/AccountInfo/AccountInfo.tsx]: oh, sorry, I misunderstood you. You are right.

**Review by OlehYavoriv — APPROVED** [review]

**Review by MeteorPro — APPROVED** [review]

**Pull request merged**